spec/unit/create.spec.js

```diff
@@ -275,6 +275,13 @@ describe('create', function () {
                 });
             });
 
+            it('should interpolate the escaped project name into strings.xml', () => {
+                config_mock.name.and.returnValue('<Incredible&App>');
```

Comment:
can we try another combination? Such as:
"& 12XPTO & W" 
"Strange $ Combinaned # App &" 
"$KUH% I*& $)OFNlkfn$"
" &amp;B&B"

Reply:
and to stir the pot, we should also add arabic characters for instance

Reply:
Well, we should just use actual valid app names for android 😅 , so take that into consideration

Reply:
Well, I don't agree :D we should not validate the "happy paths", by this I mean "a happy path is a default scenario featuring no exceptional or error conditions, so the unhappy path is all the other scenarios where exceptional or error conditions happen." and I think we should validate those unhappy paths on the unit tests.

Reply:
What do you mean with "you don't agree"? I'm just adding context of things to test 😅  When I meant the valid app names I wanted to reference to check what are valid names, and test accordingly.